Intelligent Document Service doc and Receipt Processing doc

Luyao-Zhang-1 opened the pull request:
* Intelligent Document Service (IDS)
* Invoice Processing (IP)
* Receipt Processing (RP)

Luyao-Zhang-1 committed:
Create docs for the three app services

* Intelligent Document Service (IDS)
* Invoice Processing (IP)
* Receipt Processing (RP)

Luyao-Zhang-1 commented:
The release date for these app services is set to be November 30, 2021.

Luyao-Zhang-1 commented:
This PR contains the docs for the three app services, originally from #3621

Luyao-Zhang-1 committed:
Update index.md

Commit:
updated IDS, RP documentation

Commit:
updated IDS, RP, IP documentation

Luyao-Zhang-1 committed:
Update intelligent-document.md

Luyao-Zhang-1 committed:
Merge branch 'lz-app-services-ids-ip-rp' of https://github.com/saurabhkmx/docs into pr/3779

Luyao-Zhang-1 committed:
Merge pull request #3779 from saurabhkmx/lz-app-services-ids-ip-rp

Updated IDS, RP documentation

Luyao-Zhang-1 committed:
Update format

Luyao-Zhang-1 committed:
Update receipt-processing.md

Commit:
removed powered by ABBYY. Added Technical Provider section

Commit:
updated ids, ip, ir with domain training model app detail

Luyao-Zhang-1 committed:
Merge pull request #3780 from saurabhkmx/lz-app-services-ids-ip-rp

IDS IP IR document changes

Luyao-Zhang-1 committed:
Update

Luyao-Zhang-1 committed:
Update IDS and sreenshots

Luyao-Zhang-1 committed:
Update IDS

Luyao-Zhang-1 committed:
Update IDS

Luyao-Zhang-1 committed:
Remove extra new line

Luyao-Zhang-1 committed:
Update intelligent-document.md

Luyao-Zhang-1 committed:
Update IDS

Luyao-Zhang-1 committed:
Update intelligent-document.md

Luyao-Zhang-1 committed:
Update intelligent-document.md

Luyao-Zhang-1 committed:
Update

Luyao-Zhang-1 committed:
Update RP

Luyao-Zhang-1 committed:
Update

Luyao-Zhang-1 committed:
Update IP

Luyao-Zhang-1 committed:
Update

Luyao-Zhang-1 committed:
Update

Luyao-Zhang-1 committed:
Merge branch 'development' into lz-app-services-ids-ip-rp

Luyao-Zhang-1 committed:
Fix formatting

Luyao-Zhang-1 committed:
Update

Luyao-Zhang-1 committed:
update

Luyao-Zhang-1 committed:
Update

Luyao-Zhang-1 committed:
Update docs based on feedback

Luyao-Zhang-1 committed:
Change "Import Mapping" to "import mapping"

Luyao-Zhang-1 committed:
Fix typo and todo items

Luyao-Zhang-1 committed:
Cross-reference the content in Marketplace Overview and remove the duplicate content

Luyao-Zhang-1 committed:
Remove extra lines

Luyao-Zhang-1 committed:
Update footnotes

Luyao-Zhang-1 committed:
Update footnotes

Luyao-Zhang-1 committed:
Remove extra lines

saurabhkmx commented on content/appstore/app-services/intelligent-document-service.md:
```diff
@@ -0,0 +1,177 @@
+
+    ![Create New Model dialog box](attachments/intelligent-document-service/create-new-model-dialog-box.png)
+
+4. Enter a unique **Model Name**, select a **Language**, and then click **Create Model**.
```
We want mention allowed value for Model Name. Model Name has a length limit of 255 and only character(a-z, A-Z), number(0-9) and underscore(_) values are supported. Could you please accommodate that here?

Luyao-Zhang-1 committed:
Add notes for model name

shaileshpuranikmx commented on content/appstore/app-services/intelligent-document-service.md:
```diff
@@ -0,0 +1,180 @@
+
+The [Intelligent Document Service](https://marketplace.mendix.com/link/component/118388) app service uses AI and OCR technologies<sup>1</sup> to extract text and key value pairs from images of documents in bulk.
+
+<small><sup>1</sup> The AI and OCR technologies used by Intelligent Document Service are powered by ABBYY&reg;. Application includes ABBYY® FlexiCapture® 12 SDK © 2019 ABBYY Production LLC., and also that ABBYY and FLEXICAPTURE are either registered trademarks or trademarks of ABBYY Software Ltd. and cannot be used without prior written consent of ABBYY Software Ltd.</small>
```
@mahendra-mahajan-mendix / @saurabhkmx 
For this service and invoice and receipt processing, ABBYY Lego is needed as per ABBYY legal agreement.

Rather than adding it in the Introduction section, I would add a separate section "Technology Provider" towards end and add ABBYY name, logo and trademark data.

mahendra-mahajan-mendix committed:
Update intelligent-document-service.md

Free-Trial changes for IDS services.

Luyao-Zhang-1 committed:
Merge pull request #3980 from mahendra-mahajan-mendix/patch-5

Update intelligent-document-service.md

mahendra-mahajan-mendix committed:
Update intelligent-document-service.md

further updates to the IDS documentation. Pls merge these changes after your review. i will also be sending updated screenshots, the names of the files will be same so that we dont have to update hyperlinks. Just adding the images in the directory should be needed.

Luyao-Zhang-1 committed:
Upload images

Luyao-Zhang-1 committed:
Merge pull request #3983 from mahendra-mahajan-mendix/patch-6

Update intelligent-document-service.md

Luyao-Zhang-1 committed:
Correct file name of the logo

Luyao-Zhang-1 committed:
Move images in the correct folder

Luyao-Zhang-1 committed:
Put logo in the right folder

Luyao-Zhang-1 committed:
Fix indentation

Luyao-Zhang-1 committed:
Update intelligent-document-service.md

Luyao-Zhang-1 committed:
Add border to the picture

mahendra-mahajan-mendix committed:
Update intelligent-document-service.md

Added limitation for file sizes as alerts. corrected a typo.

mahendra-mahajan-mendix committed:
Update receipt-processing.md

Changes to Receipt processing. Addition of technical provider, changes from JSON mapping to JSON structure.

mahendra-mahajan-mendix committed:
Update invoice-processing.md

Updating Invoice processing document to add "Tech provider" section. Rewording JSON Mapping file to JSON Structure and miscellaneous changes.

Luyao-Zhang-1 committed:
Merge pull request #3989 from mahendra-mahajan-mendix/patch-7

Update intelligent-document-service.md

Luyao-Zhang-1 committed:
Update intelligent-document-service.md

Luyao-Zhang-1 committed:
Merge pull request #3990 from mahendra-mahajan-mendix/patch-8

Update receipt-processing.md

Luyao-Zhang-1 committed:
Update images

Luyao-Zhang-1 committed:
Merge pull request #3991 from mahendra-mahajan-mendix/patch-9

Update invoice-processing.md

Luyao-Zhang-1 committed:
update images

mahendra-mahajan-mendix committed:
Update receipt-processing.md

Adding screenshot for existing pre-trained receipt model and minor rewording. JSON structure image needs to be taken from what is existing in the IDS documentation.

mahendra-mahajan-mendix committed:
update invoice processing doc

Adding a new screenshot for Vendor and BU information. Also changing the existing model screenshot with latest image. some rewording for free trial.

Luyao-Zhang-1 committed:
Merge pull request #4002 from mahendra-mahajan-mendix/patch-10

Update receipt-processing.md

Luyao-Zhang-1 committed:
Update receipt-processing.md

Luyao-Zhang-1 committed:
Create pre-trained-receipt-model-pub.png

Luyao-Zhang-1 committed:
Merge pull request #4003 from mahendra-mahajan-mendix/patch-11

update invoice processing doc

Luyao-Zhang-1 committed:
add images

Luyao-Zhang-1 committed:
Update new-model-status-completed.png

Luyao-Zhang-1 committed:
Update invoice processing doc

Luyao-Zhang-1 committed:
Use smaller logo

Luyao-Zhang-1 committed:
Use (even) smaller logos and add hyperlinks to ABBYY trademark

Luyao-Zhang-1 committed:
Correct names

Luyao-Zhang-1 committed:
Update image and fix typo

Luyao-Zhang-1 committed:
Merge branch 'development' into lz-app-services-ids-ip-rp

Luyao-Zhang-1 committed:
Remove image and empty lines

Luyao-Zhang-1 committed:
Add an image and add borders to images

Somehow binding-key-generation.png has been removed from the current branch. Add it back to the branch.

Luyao-Zhang-1 committed:
Add requirements for resolutions

Luyao-Zhang-1 committed:
Remove Invoice Processing doc from this PR

Luyao-Zhang-1 committed:
Fix index

Luyao-Zhang-1 commented:
The Invoice Processing app service will be released later than the Intelligent Document Service and the Receipt Processing app services. 
Therefore, I made a new PR for the Invoice Processing doc. From now on, all the work for this doc should be done in this new PR: [PR4025](https://github.com/mendix/docs/pull/4025).

Luyao-Zhang-1 committed:
Update receipt-processing.md

Luyao-Zhang-1 committed:
Update receipt-processing.md

Luyao-Zhang-1 committed:
Correct code snippet for note

Luyao-Zhang-1 committed:
Update receipt-processing.md

Luyao-Zhang-1 committed:
Correct another note code snippet

Luyao-Zhang-1 committed:
Update intelligent-document-service.md

Luyao-Zhang-1 committed:
Remove the limitations on the mark ID

Luyao-Zhang-1 committed:
update note on the limited number of users per company for trial, update SP version number

Luyao-Zhang-1 committed:
Update images for the IDS doc

Luyao-Zhang-1 committed:
Update images for the RP doc

Luyao-Zhang-1 committed:
Change wording

Luyao-Zhang-1 committed:
Remove todo items (they will be done tomorrow by the development team)

Luyao-Zhang-1 committed:
Update new-model-status-popup.png
